### DISCUSSION TITLE
RBAC: prawo do resetowania hasła kont tylko z danym UPN'em

### DISCUSSION THREAD

**Discussion by dlxpbva** [AzureCommunityPL/group#142] [Pomoc] [answered]
Cześć,

W obrębie jednego tenanta zaszła potrzeba zmigrowania ok. 1000 kont do wersji CloudOnly. Konta te będą miały zmieniony UPN na inny, niech będzie to nowy-upn.com
Czy można stworzyć rolę w Azure AD, za pomocą której mogę oddelegować prawo do resetowania hasła tylko kontom z UPN'em nowy-upn.com?
Coś jak w Exchaneg'u gdzie scope można zawęzić na zasadzie:
New-ManagementScope -RecipientRestrictionFilter "UPN -eq 'nowy-upn.com'"

**Comment by smajchrz** [comment]
Na obecną chwile nie ma takiej możliwości. Jedynie customowe uprawnienia które można nadać to te od Application registration.
</br>
`Add permissions for this custom role. Currently, permissions for Application registrations are supported in custom roles. More permissions are coming soon. Learn more `  

https://docs.microsoft.com/en-us/azure/active-directory/roles/custom-overview

**Comment by KamilWiecek** [comment, marked as answer]
Utworzenie [Azure AD Administrative unit](https://docs.microsoft.com/en-us/azure/active-directory/roles/administrative-units) w połączeniu z:
- jednorazowum wrzucenienm do adm. unit. kont *nowy-upn.com" (np. przy pomocy powershella) 
- assignmentem odpowiednij roli Azure AD dla tego adm. unit.

zaadresuje Twoje wymaganie.

**Reply by mgrabarz** [reply]
@dlxpbva czy ta odpowiedź pomogła rozwiązać problem?

**Reply by dlxpbva** [reply]
Cześć, sorry za opóźnienie w odpowiedzi.

Z technicznego punktu widzenia - tak. Niestety ograniczeniem jest dla mnie licencja AD Premium dla każdego administratora (bo w mojej sytuacji potrzeba ich kilkanaście).
